dask/array/reductions.py

```diff
@@ -466,7 +467,8 @@ def moment_chunk(A, order=2, sum=chunk.sum, numel=numel, dtype='f8', meta=False,
 
     n = n.astype(np.int64)
     total = sum(A, dtype=dtype, **kwargs)
-    u = total / n
+    with warnings.catch_warnings(record=True):
```

Comment:
I think `with np.errstate(divide='ignore', invalid='ignore'):` is preferable. This would still fail CI, as `record=True` doesn't actually filter the warnings. Would still need to add a `warnings.simplefilter('ignore')`.